### PR TITLE
Updated wordpress__viewport to 5.5

### PR DESCRIPTION
**Pull request by marekdedic**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/WordPress/gutenberg/blob/%40wordpress/viewport%405.5.0/packages/viewport/CHANGELOG.md
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wordpress__viewport ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wordpress__viewport@latest/))
was missing the following properties:
1. store
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ccbb74dd825d58534ff562f01c8a49dd58722e6c
</p>

File: types/wordpress__viewport/index.d.ts [hunk 1]
```diff
@@ -13,3 +13,11 @@ declare module '@wordpress/data' {
 
 export { default as ifViewportMatches } from './if-viewport-matches';
 export { default as withViewportMatch } from './with-viewport-match';
+
+export interface ViewportStoreDescriptor extends StoreDescriptor {
+    name: 'core/viewport';
+}
+
+declare module '@wordpress/viewport' {
```
Comment (weswigham) [inline types/wordpress__viewport/index.d.ts:21]: The containing file is `@wordpress/viewport` - can't this just be `export const store: ViewportStoreDescriptor;`?

Comment (marekdedic) [reply types/wordpress__viewport/index.d.ts:21]: Honestly, I'm not sure. What you are saying makes sense on the first look. However, I was following the example in #63988 - maybe @opr can chime in as to why he did it the way he did... :shrug:

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Please handle feedback from @weswigham

**Review by gabritto — APPROVED** [review]

**Comment by marekdedic** [comment]
Ready to merge

**Pull request merged**